package.json

```
@@ -68,5 +68,9 @@
     "tsdx": "^0.14.1",
     "tslib": "^2.3.1",
     "typescript": "^3.9.10"
+  },
+  "peerDependencies": {
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0"
```

The caret operator won't allow beyond v16, so we need to `||` additional major versions that we support in order to allow them, else v17 users will get peer dep warnings.

You'll see this pattern throughout our libs since most have `graphql` as a peer and most support a range of majors.
```suggestion
    "react": "^16.8.0 || ^17.0.0",
    "react-dom": "^16.8.0 || ^17.0.0"
```